cmd/relayproxy/main.go

```diff
@@ -56,6 +57,10 @@ func main() {
 		zapLog.Fatal("error while reading configuration", zap.Error(err))
 	}
 
+	if proxyConf == nil {
+		panic(fmt.Errorf("unable to set the proxy conf, and it is empty"))
```

Comment:
I would suggest using `zapLog.Fatal` instead, it will also close the service.

```suggestion
		zapLog.Fatal("unable to set the proxy conf, and it is empty")
```